spec/features/visitor_sees_search_bar_spec.rb

```diff
@@ -0,0 +1,32 @@
+# require 'rails_helper'
```

Comment:
If this test is no longer used please delete the file

Reply:
deleted file